src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services_instanceof_invalid_ignore.yml

```diff
@@ -0,0 +1,16 @@
+services:
+    _instanceof:
+        Symfony\Component\DependencyInjection\Tests\Fixtures\BarInterface:
+            autowire: true
+            lazy: true
+            tags:
+                - { name: foo }
+            ignore: 'Symfony\Component\DependencyInjection\Tests\Fixtures\Baz'
```

Comment:
we could make it work actually, that might be useful

Reply:
I've thought about this option, but honestly I don't have the sensibility if this is could grows the learning curve or not. In this case, I'm with your feeling. :)